tests/github/bumpVersion/functions.spec.js

```diff
@@ -0,0 +1,96 @@
+const {describe} = require('jest-circus');
```

Comment:
A few comments on this test file:

1) I would move it to `tests/unit/versionUpdaterTest.js`.
2) Not sure what the `spec.js` file extension is for, but it shouldn't be necessary.
3) I noticed you're using CJS modules in here, but in all our other tests we're using ES6 modules.

Reply:
1 - Sure!
2 - It flags a file as being a test. You may consider removing the following line from your package.json (jest options) if you are not using them: `"**/?(*.)+(spec|test).[jt]s?(x)"`
3 - I was just being a little over protective because I was not sure what I could use or not use in the .github folder.

Reply:
About 3, isn't it the same as what has been used in nativeVersionUpdaterTest?

Reply:
Oh yeah, you're right 🤦🏼‍♂️

CJS is fine for the test then.